__tests__/app.test.js

```diff
@@ -25,3 +25,38 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: should respond with the given article", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          body: expect.any(String),
```

Comment:
The use of `.toMatchObject` here is good - but on this endpoint we should be asserting for a specific value rather than in this vague manner. This is more appropriate for requesting multiple resources like your last endpoint.
For example, if your server responded with any article, would this test still pass?